client/src/components/Login/style.css

```diff
@@ -0,0 +1,96 @@
+.container {
+    width: 100%;
+    height: 100vh;
+    overflow: hidden;
+    background: #437489;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+
+.main {
+    background: transparent;
+    width: 400px;
+    height: 600px;
+    overflow: hidden;
+}
+
+.header {
+    display: flex;
+    justify-content: center;
+}
+
+.logo {
+    width: 70px;
+    margin-right: 10px;
+}
+
+.logo-text {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-family: 'Roboto', sans-serif;
+    font-size: 2rem;
+}
+
+.logo-text--white {
+    color: #fff;
+    font-weight: bold;
+}
+
+.logo-text-black {
+    color: #000;
+}
+
+.form {
+    width: 100%;
+    background: #fff;
+    height: 330px;
+    border-radius: 5px;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.label {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-evenly;
+}
+
+.label-text {
+    width: 350px;
```

Comment:
using pixels can lead to responsiveness issues. It's better to use percentages here in my opinion.

max and min width are good use cases for pixels, and we should have them to make sure if screen is too wide or too narrow the style doesn't break.